sphinx/tutorial/cylc/runtime/configuration-consolidation/index.rst

```diff
@@ -68,20 +68,23 @@ Consolidating Configuration
           [[dependencies]]
               [[[T00/PT3H]]]
                   graph = """
-                      get_observations_belmullet => gather_observations
-                      get_observations_camborne => gather_observations
-                      get_observations_heathrow => gather_observations
-                      get_observations_shetland => gather_observations
+                      get_observations_belmullet => consolidate_observations
+                      get_observations_camborne => consolidate_observations
+                      get_observations_heathrow => consolidate_observations
+                      get_observations_shetland => consolidate_observations
                   """
 
 .. nextslide::
 
 Cylc offers three ways of consolidating configurations to help improve the
 structure of a suite and avoid duplication.
 
-* Families
-* Jinja2
-* Parameterisation
```

Comment:
A note for future: as discussed in person, the 'three approaches' are now appearing on the index (left-hand) menu which is great, but upon clicking the links to go one of the pages from the section link bullets under the 'The Three Approaches' section, the relevant pages are not highlighted and also the menu collapses to the 'Runtime' level, which is not ideal. Strangely it works as desired (highlighted, with no collapse) when accessed from *this* list, though readers will usually use the other one given the flow here!